### PR TITLE
 OCM-6347 | ci: Setup rosa-hcp-pl profile

### PR DESCRIPTION
**Pull request by radtriste**
**What this PR does / why we need it**:

Enable the `rosa-hcp-pl`profile

**Which issue(s) this PR fixes** 

https://issues.redhat.com/browse/OCM-6347

**Change type**
- [ ] New feature
- [ ] Bug fix
- [ ] Build
- [x] CI
- [ ] Documentation
- [ ] Performance
- [ ] Refactor
- [ ] Style
- [ ] Unit tests
- [ ] Subsystem tests
- [ ] e2e tests

**Checklist**
- [x] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ Fixes #
 - [ ] Style
 - [ ] Unit tests
 - [ ] Subsystem tests
+- [ ] e2e tests
```
Comment (xueli181114) [inline .github/pull_request_template.md:31]: There is `CI` option. Is there any difference with  `e2e tests`?

File: tests/utils/constants/config.go [hunk 2]
```diff
@@ -119,3 +119,12 @@ func GetKubeConfigDir() string {
 	}
 	return configDir
 }
+
+func GetOutputManifestsDir() string {
```
Comment (xueli181114) [inline tests/utils/constants/config.go:123]: What is this used for?

Comment (radtriste) [reply tests/utils/constants/config.go:123]: This is a leftover of my changes ... Removed

File: tests/ci/profile_handler.go [hunk 3]
```diff
@@ -617,3 +617,11 @@ func (profile *Profile) GetClusterManifestsDir() string {
 	manifestsDir := CON.GetClusterManifestsDir(profile.GetClusterType())
 	return manifestsDir
 }
+
+func (profile *Profile) IsPrivateLink() bool {
+	if profile.GetClusterType().HCP {
+		return profile.Private
```
[the remark this answers is not in the document]
Comment (radtriste) [reply tests/ci/profile_handler.go:623]: Yes all private HCP cluster are private link.
In TF, we don't even have a `private link` field, there is only `private`.

File: .github/pull_request_template.md (hunk 1 above)
Comment (radtriste) [reply .github/pull_request_template.md:31]: `e2e tests` would be the tests cases / utils code update and `CI` would be the ci folder ?

Comment (xueli181114) [reply .github/pull_request_template.md:31]: I don't think the items indicates the folder. They should be for the updating purpose. All QE's updating can be aligned to CI. @nirarg Can you please help to confirm that?

Comment (radtriste) [reply .github/pull_request_template.md:31]: cc @gdbranco @ciaranRoche

Comment (ciaranRoche) [reply .github/pull_request_template.md:31]: +1 on QE updating to be aligned to CI

Comment (radtriste) [reply .github/pull_request_template.md:31]: ok then, will setup in `ci`

**Comment by radtriste** [comment]
@xueli181114 PR updated. Please have a look

**Comment by radtriste** [comment]
@xueli181114 @yuwang-RH Please have a final look

**Pull request merged**